### ISSUE TITLE
az image copy -  Creating target storage account - command ended with an error

### ISSUE THREAD

**Issue by sroddy5** [Azure/azure-cli-extensions#987]
- If the issue is to do with Azure CLI 2.0 in-particular, create an issue here at [Azure/azure-cli](https://github.com/Azure/azure-cli/issues)

### Extension name (the extension in question)
az image copy

### Description of issue (in as much detail as possible)
I am attempting to use az image copy however it is failing at the 'Creating target storage account' with the error message: 
```
command ended with an error: ['/opt/az/bin/python3', '-m', 'azure.cli', 'storage', 'account', 'create', '--name', 'westeuropen9q67qpahnedgr', '--resource-group', 'image-copy-rg', '--location', 'westeurope', '--sku', 'Standard_LRS', '--output', 'json', '--tags', 'created_by=image-copy-extension']"
```
Running with --debug produces no additional error information.

Here is my input, source is in the uswest2 region
```
az image copy --source-resource-group {}--source-object-name {} --target-location westeurope --target-resource-group {}
```

Enviroment:
Azure Portal Cloud Shell
sean@Azure:~$ az --version
azure-cli                         2.0.74

command-modules-nspkg              2.0.3
core                              2.0.74
nspkg                              3.0.4
telemetry                          1.0.3

Extensions:
image-copy-extension               0.2.1

Python location '/opt/az/bin/python3'
Extensions directory '/home/sean/.azure/cliextensions'

Python (Linux) 3.6.5 (default, Sep 20 2019, 05:42:51)
[GCC 5.4.0 20160609]

-----

**Comment by tamirkamara** [comment]
A recent change in the azure cli core returns a response in an invalid json format which breaks the extension. This is documented in this github issue: #azure-cli/10687
It will take some time until we have a good approach and do the necessary changes. In the meantime, I can suggest downgrading the cli to version 2.0.72 or below which should function as before.

**Comment by tamirkamara** [comment]
I believe this issue is resolved with version 0.2.2 of the extension.
@zikalino please close it.

**Comment by tamirkamara** [comment]
@qwordy this is fixed I think. can you close it?
